src/pages/Environments.js

```diff
@@ -334,13 +334,12 @@ const Environments = () => {
             }
           },
           {
-            size: { basis: 90, grow: 0 },
+            size: { basis: 100, grow: 0 },
             headerRenderer: () => h(Sortable, { sort, field: 'created', onSort: setSort }, ['Type']),
             cellRenderer: ({ rowIndex }) => {
               const cloudEnvironment = filteredCloudEnvironments[rowIndex]
-              // TODO: update return logic once we support more app types (will need a backend change to return appType in list apps endpoint as well)
               return cloudEnvironment.appName ?
-                'Galaxy' :
+                (cloudEnvironment.appType ? _.capitalize(cloudEnvironment.appType) : 'Galaxy') :
                 (cloudEnvironment.runtimeConfig.cloudService === 'DATAPROC' ? 'Dataproc' : cloudEnvironment.runtimeConfig.cloudService)
```

Comment:
@kyuksel I see that you added the above line. Is there any reason to not just do _.capitalize(cloudEnvironment.runtimeConfig.cloudService)? I'm using the capitalize function to convert from CROMWELL to Cromwell.

(I don't know though what cloudEnvironment.runtimeConfig.cloudService will be if it is **not** DATAPROC)

Reply:
@kyuksel but would you want "GCE" to be "Gce"? I would imagine not (which means I can't just always use capitalize).

Reply:
Ah yea good point! I have some ideas around how we may be able to better handle string cases by categorizing them as:

1. coming from API responses/input into API requests
2. used internally
3. displayed to users

and providing utilities to make conversions where necessary…but this could be a discussion for another time (perhaps the next Terra UI meeting :)